robotbase/src/main/java/frc/robot/state/RobotState.java

```diff
@@ -0,0 +1,93 @@
+package frc.robot.state;
+
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+
+public class RobotState {
+  private static final RobotState s_instance = new RobotState();
```

Comment:
I would not do a singleton, just do a static instance in Robot.java like we are doing for the subsystems. That way we stay consistent.